applications/DamApplication/custom_elements/small_displacement_element.cpp

```diff
@@ -0,0 +1,457 @@
+//
```

Comment:
I recommend to fully migrate to the Structural elements to avoid code duplication

Reply:
Agree, the same formulation is also implemented in the `StructuralMechanicsApplication`: `SmallDisplacement`.

Reply:
I preserved the old Solid elements for retro-compatibility but I exported them with a different name from the one used in the Structural to avoid conflicts.